plaso/parsers/google_logging.py

```diff
@@ -0,0 +1,264 @@
+"""Parser for Google-formatted log files."""
+
+from __future__ import unicode_literals
+
+import re
+
+from dfdatetime import time_elements as dfdatetime_time_elements
+
+import pyparsing
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import errors
+from plaso.lib import definitions
+from plaso.parsers import manager
+from plaso.parsers import text_parser
+
+
+
+class GoogleLogEventData(events.EventData):
+  """Convenience class for events from a Google log file.
```

Comment:
Please change to "Google log file event data." 

"Convenience class"  has a specific meaning https://en.wiktionary.org/wiki/convenience_class

Did you mean "value object" https://en.wikipedia.org/wiki/Value_object ?

Reply:
Done